frontend/packages/ceph-storage-plugin/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@console/ceph-storage-plugin",
+  "version": "0.0.0-fixed",
+  "description": "Ceph Storage - Persistent storage for kubernetes ",
+  "private": true,
+  "dependencies": {
+    "@console/plugin-sdk": "0.0.0-fixed",
+    "@console/shared": "0.0.0-fixed"
+  },
+  "consolePlugin": {
+    "entry": "src/plugin.tsx"
```

Comment:
Unless you need to declare React components in your plugin's entry module, I'd use `.ts` instead of `.tsx`.

It's a good practice to put all of your React components into separate modules (i.e. under `src/components`) and import & use them in the plugin entry module. This helps to keep your plugin entry small and clean.